docs/en_US/network-dumps.md

```diff
@@ -0,0 +1,137 @@
+
+Full credit to the upkeep of the repository, and creation of the original Docker container, goes to GitHub user [MatthewL246](https://github.com/MatthewL246).
+
+Install Docker for your operating system using the official [setup guide](https://docs.docker.com/get-docker/). Then follow the steps for your console type.
+
+## 3DS
+1. Download [this IPS patch](https://github.com/PretendoNetwork/mitmproxy-nintendo/raw/master/ssl-patches/0004013000002F02.ips). This IPS patch patches the SSL sysmodule to disable SSL verification, allowing the console to connect to the proxy server with TLS connections.
+2. Place the patch on your SD card at `SD:/luma/sysmodules/0004013000002F02.ips`.
+3. Place the SD card back into your 3DS.
+4. Launch into the Luma settings by holding `SELECT` while powering on.
+5. Ensure both `Enable loading external FIRMS and modules` and `Enable game patching` are enabled before booting.
+6. Launch Nimbus and connect to Nintendo Network.
+7. On your computer, create a `3ds-dumps` directory and run the following command in a command prompt: `docker run -it --rm -p 8083:8083 -v ./3ds-dumps:/home/mitmproxy/dumps ghcr.io/pretendonetwork/mitmproxy-nintendo:3ds mitmdump`
+8. This command starts the proxy server using Docker, exposing the servers port 8083 on your computers port also on 8083, and links the `/home/mitmproxy/dumps` directory in the container to the `3ds-dumps` directory you just created.
+9. On your 3DS, launch into Internet Settings and select your connection.
+10. Select `Change Settings > Proxy Settings`.
+11. Select `Yes` and then `Detailed Setup`.
+12. Enter your computers local IP address into `Proxy Server` and 8083 into `Port`.
+13. Select `Ok` then `Save` and run the connection test. Your 3DS should connect to the internet and you should see connections being made in the proxy server
+14. See the end of this section for final steps.
+
+## Wii U
+1. Download [this Aroma setup module](https://github.com/PretendoNetwork/mitmproxy-nintendo/raw/master/ssl-patches/30_nossl.rpx). This patches the SSL sysmodule to disable SSL verification, allowing the console to connect to the proxy server with TLS connections.
+2. Place the patch on your SD card at `SD:/wiiu/environments/aroma/modules/setup/30_nossl.rpx`. If there are other patches with the same ID `30`, this is fine.
+3. Place the SD card back into your Wii U and turn on the console.
+4. Launch into the Aroma settings by pressing `L + DPAD-DOWN + SELECT`.
+5. Enter `Inkay > Patching` and toggle `Connect to the Pretendo Network` to ***FALSE***.
+6. On your computer, create a `wiiu-dumps` directory and run the following command in a command prompt: `docker run -it --rm -p 8082:8082 -v ./wiiu-dumps:/home/mitmproxy/dumps ghcr.io/pretendonetwork/mitmproxy-nintendo:wiiu mitmdump`
```

Comment:
When using the Windows command line, the volume needs to be `-v .\wiiu-dumps:/home/mitmproxy/dumps` (with the backslash for the source path). Alternatively, you could tell Windows users to run `wsl docker run -it --rm -p 8082:8082 -v ./wiiu-dumps:/home/mitmproxy/dumps ghcr.io/pretendonetwork/mitmproxy-nintendo:wiiu mitmdump` to run the command inside WSL, where the forward slash bind mount does work.

Reply:
Is WSL installed and enabled by default on Windows? These docs are aiming to cater to brand new users who may have never touched even a command prompt before. I've had to hold peoples hands while they installed software like Fiddler, which has an installation wizard, so I'd like to avoid overwhelming these users

Reply:
No, it's not enabled by default. The thing is, Docker Desktop for Windows [requires WSL 2 to be set up](https://docs.docker.com/desktop/install/windows-install/) because the Docker daemon itself runs inside WSL. So, anyone with Docker installed on Windows will necessarily have WSL2 enabled.

Reply:
> Docker Desktop for Windows [requires WSL 2 to be set up](https://docs.docker.com/desktop/install/windows-install/)

not necessarily, there is another option for using the Hyper-V backend, which presumably doesn't use WSL? haven't tested though

Reply:
I think WSL was the default option when I installed it. Unfortunately installing WSL2 and Docker Desktop for Windows might require some hand-holding.

Reply:
> > Docker Desktop for Windows [requires WSL 2 to be set up](https://docs.docker.com/desktop/install/windows-install/)
> 
> not necessarily, there is another option for using the Hyper-V backend, which presumably doesn't use WSL? haven't tested though

The Hyper-V backend is for running Windows containers I thought? Since this runs a Linux container users would likely still need the WSL backend. I'll update the docs to clarify that users need the WSL version